bigbluebutton-html5/private/locales/en.json

```diff
@@ -354,6 +354,10 @@
     "app.audioModal.settingsTitle": "Change your audio settings",
     "app.audioModal.helpTitle": "There was an issue with your media devices",
     "app.audioModal.helpText": "Did you give permission for access to your microphone?  Note that a dialog should appear when you try to join audio, asking for your media device permissions, please accept that in order to join the audio conference.  If that is not the case, try changing your microphone permissions in your browser's settings.",
+    "app.audioModal.audioDialTitle": "Join the audio using your phone",
+    "app.audioModal.audioDialLabel": "Join the audio using your phone ➔",
```

Comment:
Not sure why you need two separate strings here. Could you concatenate with the arrow in the code? This would be tricky to translate or for the screen reader to spell out